webkit_server.py

```diff
@@ -513,12 +520,47 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
-      self._writeline(str(len(arg)))
-      self._sock.sendall(arg.encode("utf-8"))
+        self.send_arg(arg)
```

Comment:
Thanks, I've fixed it in this commit https://github.com/niklasb/webkit-server/pull/33/commits/62ad2e9b233e03ad45403bc717fa4233ca1a10df.